api/v1beta1/disruption_webhook.go

```diff
@@ -508,3 +520,30 @@ func safetyNetNeitherHostNorPort(r Disruption) bool {
 
 	return false
 }
+
+// safetyNetDiskFailurePath is the safety net regarding missing path or invalid path values for a disk failure disruption.
+func safetyNetDiskFailurePath(r *Disruption) (bool, string) {
+	if r.Spec.Unsafemode != nil && r.Spec.Unsafemode.DisableDiskFailurePath {
+		return false, ""
+	}
+
+	if r.Spec.DiskFailure.Path == "" {
```

Comment:
If the failure path _must_ be specified always, or it won't work, this should go into Validate, rather than safe mode?

Reply:
Indeed, this is pertinent! 👍  I will remove this check from the safe mode.